src/ucs/datastruct/ptr_map.inl

```diff
@@ -131,6 +131,7 @@ ucs_ptr_map_extract(ucs_ptr_map_t *map, ucs_ptr_map_key_t key)
     void *value;
 
     if (ucs_likely(!(key & UCS_PTR_MAP_KEY_INDIRECT_FLAG))) {
+        ucs_assert(key != 0);
```

Comment:
why? NULL is also pointer, kind of...
if this happens unexpectedly, IMO, it should be fixed/asserted in UCP

Reply:
why would you want to put a null pointer into the ptr_map..?

Reply:
IMO NULL pointer is incorrect usage (we expect an id to resolve to a valid object), and ptr_map can help by catching this early
+ can add assert when adding element

Reply:
I don't :) but IMHO it should not be restricted in UCS
anyway let's do it symmetric - if assert then for both add/del